httpserver/src/main/kotlin/tbdex/sdk/httpserver/models/ExchangesApi.kt

```diff
@@ -1,6 +1,12 @@
 package tbdex.sdk.httpserver.models
 
-import tbdex.sdk.protocol.models.*
+import tbdex.sdk.protocol.models.Close
```

Comment:
my import auto format must be different from other folks :(

Reply:
it is better! I couldn't figure out how to make mine do explicit imports, think it has something to do with my detekt settings being fukt